lib/steep/drivers/worker.rb

```diff
@@ -21,7 +21,7 @@ def initialize(stdout:, stderr:, stdin:)
 
       def run()
         Steep.logger.tagged("#{worker_type}:#{worker_name}") do
-          project = load_config()
+          project = load_config(warnings: false) # Main process has already emitted any warnings
```

Comment:
See https://github.com/soutaro/steep/pull/968/files#r1403807513

The warning systems of these two PRs may conflict.

Reply:
Actually, these two aren’t the only places where Steep LSP crashes, despite having opportunities to fail safely.
At least These two are (likely) the easiest to trip over.